### PR TITLE
Indexfix

### PR DESCRIPTION
**Pull request by shantanugupta-yb**

### CODE REVIEW

**Review by shaharuk-yb — CHANGES_REQUESTED** [review]
Can we rename dir yb_regular to yugabyte to have it consistent across all folders. Existing folders use, yugabyte.

need to remove use_dist_in_explain=true in some postgres yamls. It is not supported in postgres

File: config/yugabyte/orderby_workloads/postgres/ORDG5_pg_impact_of_datatype_with_sec_ind_desc.yaml [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+url: jdbc:postgresql://{{endpoint}}:5432/postgres?sslmode=require
+username: {{user}}
+password: {{pwd}}
+batchsize: 128
+isolation: TRANSACTION_REPEATABLE_READ
+loaderthreads: 1
+terminals: 1
+collect_pg_stat_statements: true
+yaml_version: v1.0
+use_dist_in_explain : true
```
[the remark this answers is not in the document]
Comment (shantanugupta-yb) [reply config/yugabyte/orderby_workloads/postgres/ORDG5_pg_impact_of_datatype_with_sec_ind_desc.yaml]: Removed

File: config/yugabyte/orderby_workloads/postgres/ORDG6_pg_impact_of_increasing_rows.yaml [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+url: jdbc:postgresql://{{endpoint}}:5432/postgres?sslmode=require
+username: {{user}}
+password: {{pwd}}
+batchsize: 128
+isolation: TRANSACTION_REPEATABLE_READ
+loaderthreads: 1
+terminals: 1
+collect_pg_stat_statements: true
+yaml_version: v1.0
+use_dist_in_explain : true
```
[the remark this answers is not in the document]
Comment (shantanugupta-yb) [reply config/yugabyte/orderby_workloads/postgres/ORDG6_pg_impact_of_increasing_rows.yaml]: Removed

**Review by hbhanawat — APPROVED** [review]
Looks good. Please fix Shaharuk's comments before merging.

**Review by shaharuk-yb — APPROVED** [review]
LGTM

**Pull request merged**